scripts/process.zs

```diff
@@ -103,7 +103,7 @@ function sawWood(input as IIngredient, output as IItemStack, exceptions as strin
 function crush(input as IIngredient, output as IItemStack, exceptions as string = null, extra as IItemStack[] = null, extraChance as float[] = null, opts as IData = null) {
   for machine in [
     'Macerator', 'eu2Crusher',
-    'AACrusher', 'IECrusher', 'SagMill',
+    'AACrusher', 'IECrusher', 'SagMill', 'massspectrometer',
```

Comment:
Adding MassSpectrometer to `crush()` function os bad idea:
1. Mass Spectrometry is not "crushing" materials - its using maunets to pull one atoms from others.
2. Adding MS there will cause it working for any crush() operation, even if its just Coal => Coal Dust crushing.

You need to use Mass Spectrometer separately - it does not apply to any existing process functions.